execution/record.go

```diff
@@ -179,7 +179,7 @@ func (r *Record) Show() string {
 		parts[i] = fmt.Sprintf("%s: %s", field.Name, r.Value(field.Name).Show())
 	}
 
-	return fmt.Sprintf("{%s}", strings.Join(parts, ", "))
+	return fmt.Sprintf("{%s, %s}", r.Metadata.Id, strings.Join(parts, ", "))
```

Comment:
If you add the record ID to the show, at least add it in a form as above. Make r.ShowFields() return sys.id, and handle this in Value() to return the id.

Reply:
Don't special case it.